test/Reflection/ReflectionClassTest.php

```diff
@@ -18,6 +18,12 @@ private function getComposerLocator()
         return new ComposerSourceLocator($loader);
     }
 
+    /**
+     * @covers \BetterReflection\Reflection\ReflectionClass::inNamespace()
```

Comment:
@asgrim if you add private methods, you start lagging back. The SUT is the class per-se in this sort of test, so I wouldn't go this far, mainly to avoid having to constantly update annotations as internal refactoring occurs